fix_formatting.py

```
@@ -0,0 +1,639 @@
+#!/usr/bin/env python3
+"""
+Script to fix formatting issues in Python files.
+"""
+
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)